PushSharp.Apple/AppleHttpPushChannelSettings.cs

```diff
@@ -0,0 +1,32 @@
+﻿using PushSharp.Core;
+
+namespace PushSharp.Apple
+{
+    public class AppleHttpPushChannelSettings : IPushChannelSettings
+    {
+        public const string BundleId = "com.pof.mobileapp.iphone";
```

Comment:
Specific BundleId should _not_ be set in the library

Reply:
moved to endpoint config